src/mixins/embed.js

```diff
@@ -1,4 +1,8 @@
-import * as cookie from 'tiny-cookie';
+import {
+  isCookieEnabled,
+  getCookie,
+  setCookie,
+} from 'tiny-cookie';
```

Comment:
I think we have used `tiny-cookie` in optimize exp too, can you plz update that part also?